roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_rhel_kvm/assert-configuration.yml

```diff
@@ -18,9 +20,10 @@
   ignore_errors: "{{ sap_hypervisor_node_preconfigure_ignore_failed_assertion }}"
 
 - name: "Assert - Kernel same page merging (KSM) Tuning Daemon: Get status"
-  ansible.builtin.shell: systemctl status ksmtuned
+  ansible.builtin.command: systemctl status ksmtuned
```

Comment:
Same

Reply:
I would like to leave the rhel_kvm code as it is for now, but I'll keep that in my backlog.

Reply:
Same as above